modules/library/render/src/test/java/org/geotools/renderer/lite/LetterConflictTest.java

```diff
@@ -0,0 +1,300 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2011 - 2015, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.renderer.lite;
+
+import junit.framework.TestCase;
+import org.geotools.data.property.PropertyDataStore;
+import org.geotools.data.simple.SimpleFeatureSource;
+import org.geotools.geometry.jts.ReferencedEnvelope;
+import org.geotools.map.DefaultMapContext;
+import org.geotools.referencing.crs.DefaultGeographicCRS;
+import org.geotools.renderer.label.LabelCacheImpl;
+import org.geotools.styling.Style;
+import org.geotools.test.TestData;
+
+import javax.imageio.ImageIO;
+import java.awt.*;
+import java.awt.event.WindowAdapter;
+import java.awt.event.WindowEvent;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.awt.RenderingHints.KEY_ANTIALIASING;
+import static java.awt.RenderingHints.VALUE_ANTIALIAS_ON;
+
+/**
+ * Created by mmichaud on 19/08/2015.
+ */
+public class LetterConflictTest extends TestCase {
+
+    private boolean SHOW_RESULT = Boolean.getBoolean("org.geotools.labelcache.showLetterConflictTestResult");
+    private boolean OUTPUT_IMAGE = Boolean.getBoolean("org.geotools.labelcache.outputLetterConflictTestResult");
+
```

Comment:
You seem to have reverse engineered our -P interactive.image setting, but have used a different system property name.

For more info http://docs.geotools.org/latest/userguide/build/maven/testing.html